src/spec-node/featuresCLI/package.ts

```diff
@@ -113,5 +113,8 @@ async function featuresPackage({
 	const exitCode = await doFeaturesPackageCommand(args);
 
 	await dispose();
-	process.exit(exitCode);
+
+	if (shouldExit) {
```

Comment:
Code duplication for [most of this part ](https://github.com/devcontainers/cli/blob/main/src/spec-node/featuresCLI/package.ts#L70-L102) gets avoided if I use `featuresPackage` instead of `doFeaturesPackageCommand`.

Reply:
If it is an unnatural split, feel free to refactor it.  I think i'd prefer refactoring over unnecessary parameters to alter whether it exits or not

Reply:
Refactored! 👍